test/e2e.test.ts

```diff
@@ -124,6 +142,19 @@ describe('E2E', () => {
         })
       ));
 
+  it('read', () =>
```

Comment:
The test name should be updated here.

Reply:
Done, but I think we should normalize the descriptions at some points. E.g. always like `it('Should ...')`.

Reply:
Agree